server/test-compiler.js

```diff
@@ -2,13 +2,17 @@
 
 var fs = require('fs');
 var tmp = require('tmp');
+var Bluebird = require('bluebird');
 
-function createCompilationFile(body, done) {
-  tmp.file(function (err, path, fd) {
-    fs.writeFile(path, compile(body), function () {
-      done(path);
-    });
-  });
+Bluebird.promisifyAll(fs);
+Bluebird.promisifyAll(tmp);
+
+function createCompilationFile(body) {
+  return Bluebird.join(tmp.fileAsync().get(0), compile(body), writeContent);
+}
```

Comment:
Could we rename the import to `Promise` instead of `Bluebird`?

Bluebird says nothing to me, and if we change latter change the implementation, it gets outdated. 

On the other hand, Promise tells a lot to me since it is a common constrol structure. 

Looking at the docs, it seems that Bluebird also calls them that way: https://github.com/petkaantonov/bluebird/blob/master/API.md